src/communication/azext_communication/generated/_help.py

```diff
@@ -0,0 +1,125 @@
+    examples:
+      - name: List by resource group
+        text: |-
+               az communication service list --resource-group "MyResourceGroup"
+"""
+
+helps['communication service show'] = """
+    type: command
+    short-summary: "Get the CommunicationService and its properties."
+    examples:
+      - name: Get resource
+        text: |-
+               az communication service show --name "MyCommunicationResource" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service create'] = """
+    type: command
+    short-summary: "Create a new CommunicationService or update an existing CommunicationService."
+    examples:
+      - name: Create or update resource
+        text: |-
+               az communication service create --name "MyCommunicationResource" --location "Global" --data-location \
+"United States" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service update'] = """
+    type: command
+    short-summary: "Operation to update an existing CommunicationService."
+    examples:
+      - name: Update resource
+        text: |-
+               az communication service update --name "MyCommunicationResource" --tags newTag="newVal" \
+--resource-group "MyResourceGroup"
+"""
+
+helps['communication service delete'] = """
+    type: command
+    short-summary: "Operation to delete a CommunicationService."
+    examples:
+      - name: Delete resource
+        text: |-
+               az communication service delete --name "MyCommunicationResource" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service link-notification-hub'] = """
+    type: command
+    short-summary: "Links an Azure Notification Hub to this communication service."
+    examples:
+      - name: Link notification hub
+        text: |-
+               az communication service link-notification-hub --name "MyCommunicationResource" --connection-string \
+"Endpoint=sb://MyNamespace.servicebus.windows.net/;SharedAccessKey=abcd1234" --resource-id \
+"/subscriptions/12345/resourceGroups/MyOtherResourceGroup/providers/Microsoft.NotificationHubs/namespaces/MyNamespace/n\
+otificationHubs/MyHub" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service list-key'] = """
+    type: command
+    short-summary: "Get the access keys of the CommunicationService resource."
+    examples:
+      - name: List keys
+        text: |-
+               az communication service list-key --name "MyCommunicationResource" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service regenerate-key'] = """
+    type: command
+    short-summary: "Regenerate CommunicationService access key. PrimaryKey and SecondaryKey cannot be regenerated at \
+the same time."
+    examples:
+      - name: Regenerate key
+        text: |-
+               az communication service regenerate-key --name "MyCommunicationResource" --key-type "Primary" \
+--resource-group "MyResourceGroup"
```

Comment:
what about adding these commands in one new command group like `az storage account keys`?